docs/getting_started/faq.rst

```diff
@@ -22,13 +22,28 @@ A: This means that either you provided the wrong password and/or the account
 you are trying to sign in with has 2FA enabled, and as such, either needs a 2FA
 token or a refresh token to sign in. A refresh token is preferred, because then
 you will not need to enter a 2FA token in order to sign in, and the session
-will last for longer than an hour. Refer to :ref:`2FA` and :ref:`code_flow` in
-order to use the respective auth methods.
+will last for longer than an hour. Refer to :ref:`2FA` and :ref:`refresh_token`
+in order to use the respective auth methods.
 
 .. _faq3:
 
 Q: Some options (like getting moderator logs from r/mod) keep on timing out.
 How can I extend the timeout?
 
 A: Set the timeout config option or initialize :class:`.Reddit` with a timeout
-of your choosing.
\ No newline at end of file
+of your choosing.
+
+.. _faq4:
+
+Q: Help, I keep on getting redirected to `/login`!
```

Comment:
This would be a lot more searchable (which IMO is a goal for FAQs) if you include the exact exception text somewhere.

Reply:
There's a lot of things that can cause a redirect. Anything with Multireddit editing, trying to view upvoted, subreddit data, even as simple as trying to upvote

Reply:
What I'm saying is that this section should include the text "Redirect: Redirect to /r/subreddit/login/ (You may be trying to perform a non-read-only action via a read-only instance.)" (which I got from [your comment](https://github.com/praw-dev/praw/issues/1448#issuecomment-620185217))

Reply:
So far, it seems that only subreddit moderation actions redirect to login.

Reply:
Maybe something like
```suggestion
Q: Help, I keep on getting redirected to ``/r/subreddit/login/``!
```

Reply:
I don't care about what exactly the title of the question is. I want the full exception text to be on the page because searching for exception text is a very common debugging method.

If you wanted, it could be part of the answer, or perhaps an elaboration on the question in a new paragraph.